pkgs/swarmauri/swarmauri/image_gens/concrete/BlackForestImgGenModel.py

```diff
@@ -0,0 +1,259 @@
+import httpx
+import time
+from typing import List, Literal, Optional, Dict, ClassVar
+from pydantic import PrivateAttr
+from swarmauri.utils.retry_decorator import retry_on_status_codes
+from swarmauri.image_gens.base.ImageGenBase import ImageGenBase
+import asyncio
+import contextlib
+
+
+class BlackForestImgGenModel(ImageGenBase):
+    """
+    A model for generating images using FluxPro's image generation models through the Black Forest API.
+    Link to API key: https://api.bfl.ml/auth/profile
+    """
+
+    _BASE_URL: str = PrivateAttr("https://api.bfl.ml")
+    _client: httpx.Client = PrivateAttr()
+    _async_client: httpx.AsyncClient = PrivateAttr(default=None)
+
+    api_key: str
+    allowed_models: List[str] = ["flux-pro-1.1", "flux-pro", "flux-dev"]
+
+    asyncio: ClassVar = asyncio
+    name: str = "flux-pro"  # Default model
+    type: Literal["BlackForestImgGenModel"] = "BlackForestImgGenModel"
+
+    def __init__(self, **data):
+        """
+        Initializes the BlackForestImgGenModel instance with HTTP clients.
+        """
+        super().__init__(**data)
```

Comment:
change `**data` to `**kwargs`

move `super().__init__(**kwargs)` to the end of this method

Reply:
Okay. Noted!. 
Also i will be moving all Image Gen Models, to the `image_gens/concrete/`